File: verybigsum.java
import java.util.*;

public class Solution {

    public static long aVeryBigSum(long[] ar) 
    {
        long total = 0;
        for (long num : ar) 
        {
            total += num;
        }
        return total;
    }
    public static void main(String[] args) 
    {
        Scanner scanner = new Scanner(System.in);
        int n = scanner.nextInt();
        long[] ar = new long[n];
        for (int i = 0; i < n; i++) 
        {
            ar[i] = scanner.nextLong();
        }
        long result = aVeryBigSum(ar);
        System.out.println(result);
        scanner.close();
    }
}

File: CSHARP/verybigsum.cs
using System;
using System.Linq;

class solution
{
    static long aVeryBigSum(long[] ar)
    {
        long total=0;
        foreach(long num in ar)
        {
            total += num;
        }
        return total;
    }
        static void Main(string[] args)
        {
            int arCount = Convert.ToInt32(Console.ReadLine());
            long[] ar =Array.ConvertAll(Console.ReadLine().Split(' '),long.Parse);;
            long result = aVeryBigSum(ar);
            Console.WriteLine(result);
        }
}